app/src/main/cpp/skyline/common.h

```diff
@@ -375,6 +378,7 @@ namespace skyline {
         std::shared_ptr<NCE> nce; //!< This holds a reference to the NCE class
         std::shared_ptr<gpu::GPU> gpu; //!< This holds a reference to the GPU class
         std::shared_ptr<audio::Audio> audio; //!< This holds a reference to the Audio class
+        std::shared_ptr<loader::Loader> loader; //!< This holds a reference to the Loader class
```

Comment:
Consider if putting this in `OS` would be a better idea than having it here, if not then keeping it here is fine too.

Reply:
I don't really see much benefit over state, so it's probably best to keep where it is